packages/contracts/test/tcr/registryWithAppeals/grantAppeal.ts

```diff
@@ -13,6 +13,87 @@ const PLCRVoting = artifacts.require("PLCRVoting");
 const NEWSROOM_NAME = "unused newsroom name";
 
 contract("Registry With Appeals", accounts => {
+  describe("Function: setAppealFee", () => {
```

Comment:
yeah I think these should each go into their own file. 1 file per function that you're testing, basically. Besides that, they look great and I'll approve it when that's done.